### PR TITLE
fix: update Nonce verification error message

### LINKED ISSUE #4129: fix: remove the word "nonce" from all front end error messages.
# Bug Report

## User Story

As a donor or site admin, I want error messages that are helpful and inoffensive so that I can take action to resolve the problem.

In the UK (at least) the word `nonce` is slang associated with concept of pedophilia.

## Current Behavior
There are several places in the Give Core Code as well as add-ons that the word `nonce` is used in the front-end output of an error. So in addition to it not being helpful (a donor has no idea what `Nonce verification failed` means or the steps to resolve the problem), it can be offensive to see a slang term displayed.

## Expected Behavior
I expect any error message to point to what specifically is going wrong, and how the reader of the error message can resolve the issue. 

I expect that a site administrator or support technician would know exactly where in the code the error message is coming from to be able to resolve it quickly.

I expect there to be no offensive terminology in error messages.
## Bug Type
- [x] This bug describes functionality that never worked as expected.

## Steps to Reproduce
1. Browse the code searching for the word `nonce` in translatable/echoed strings. 
2. See that many of them are similar to one another, but slightly different in terms of precisely the same string.

## Possible Solution
Make a list of each error message, and explain in layman's terms what is actually happening to cause the error message.
Create error messages that allow the Donor, User, and Support technician to all take necessary action.

## Related
https://secure.helpscout.net/conversation/873431370/41330/

## Acceptance Criteria
- [ ] the word `nonce` is never output on the front end or back end of the site.
- [ ] All error messages have a unique identifier, eg `Error 38: session issues. Please contact _______`
- [ ] All error messages have a next step for the reader of the message.

### PR DESCRIPTION
**Pull request by mathetos**
## Description

Fixes #4129 to remove the word "nonce" from appearing. This word is considered vulgar in the UK. Additionally, we have formatted these error messages in four different ways. This consolidates them all into one consistent string format to make the job of translating easier for translators. 

## How Has This Been Tested?

Visual confirmation.

## Types of changes

Various strings updated. *NOTE:* `npm run dev` was NOT run, so the `POT` files will be generated later.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

**Comment by Benunc** [comment]
This still doesn't tell me as a support tech where the problem is occurring. Since the fix is different for nonce errors on different spots, I'd love something distinguishing them from one another in the error message, or at least in a log.

**Comment by mathetos** [comment]
From my quick look at the context of each string, most of these are generic error helper functions; meaning the same error is used in many different places -- but most often for only slightly different reasons. Truthfully, I don't think there's much more actionable information to be provided, if a session fails, whether it's in the admin or frontend it's simply that the session couldn't identify the user, there's not much nuance to it.
